pkg/controllers/jobset_controller.go

```diff
@@ -43,6 +43,18 @@ import (
 const (
 	RestartsKey    string = "jobset.sigs.k8s.io/restart-attempt"
 	maxParallelism int    = 50
+
+	// Event reasons and messages
```

Comment:
If these constants keep growing, maybe we should consider moving them out of this file.

Its mostly a nit.

Reply:
I agree, moved to a constants file